backend/courses/views.py

```diff
@@ -376,3 +385,130 @@ def get_queryset(self):
             section__course__semester=get_current_semester(),
             in_add_drop_period=True,
         ).order_by("created_at")
+
+def get_accepted_friends(user):
+    """ Return user's accepted friends"""
+    Friendship.objects.filter(
+        Q(sender=user) | Q(recipient=user), 
+        status=Friendship.Status.ACCEPTED
+    ).values()
```

Comment:
Let's actually make this return a queryset (tweaked the usage in `courses/views` so that's more convenient). So in other words we can remove `.values()`.